### ISSUE TITLE
Nested components trigger the "invalid element name" rule

### ISSUE THREAD

**Issue by Ambrosia** [rstacruz/stylelint-rscss#9]
Hi,

[This page](http://rscss.io/layouts.html#define-positioning-in-parents) advises you to set a component's positioning in the parent component. However, actually doing this triggers the invalid element name rule.

Perhaps it would be a good idea to stop this and instead add a new rule that triggers if the nested component applies the positioning properties not mentioned [here](http://rscss.io/layouts.html#avoid-positioning-properties) (with a different error).

**Comment by fiskhandlarn** [comment]
+1!

**Comment by lederer** [comment]
A workaround is to [configure the element rule](https://github.com/rstacruz/stylelint-rscss/blob/master/docs/rules.md) in `.stylelintrc` to use a regexp that accepts both component and element names. Eg:

```
"rules": {
    "rscss/class-format": [
        true,
        {
            "element": "^([a-z][a-z0-9]*)(-[a-z0-9]+)*$"
        }
    ]
}
```
